### ISSUE TITLE
Allow sorting of people  albums if list is long

### ISSUE THREAD

**Issue by cbenning** [cbenning/fussel#15]
Should allow sorting, as well as go straight to starting letter perhaps

**Comment by miek770** [comment]
Hi @cbenning, what's the current album sorting method, by title (i.e.: directory name)?

**Comment by cbenning** [comment]
@miek770 yeah right now it is just whatever order the JSON is built in and the python code iterates alphabetically so it ends up being alphabetical. I was hoping to build a front-end sorting and searching mechanism once I get a little spare time.
